### DISCUSSION TITLE
Mqtt - Nuova definizione entità

### DISCUSSION THREAD

**Discussion by Ale1975-Iron** [indomus/forum#1930] [HUB: Home Assistant]
Ciao a tutti, potrà essere una domanda banale e scontata per molti... mi piacerebbe sapere se c'è un metodo per mantenere ancora tutto "ordinato e separato" una volta che arriverà la versione 2022.12.0.
al momento in configuration.yaml avevo gestito tutto come segue:
```
### Automazioni ###
automation: !include automations.yaml
### Sensori Binari ###
binary_sensor: !include binary-sensors.yaml
### Camera ###
camera: !include cameras.yaml
### Contatori ###
counter: !include counters.yaml
### Fan ###
fan: !include fans.yaml
### Gruppi ###
group: !include groups.yaml
### Luci ###
light: !include lights.yaml
### Scene ###
scene: !include scenes.yaml
### Sensori ###
sensor: !include sensors.yaml
### Interruttori ###
switch: !include switches.yaml
```
con la modifca dell'istanza Mqtt c'è un modo per mantenere le tipologie di entità divise o tutto andrà elencato solo al suo interno?

Ciao e Grazie

**Comment by snakuzzo** [comment]
Io ti consiglio di lasciar perdere i file sidecar ed utilizzare i package. In questo modo te le ordini come ti pare in base alle tue esigenze

**Reply by Ale1975-Iron** [reply]
Grazie! Spero di non combinare casini ;-)

**Reply by Ale1975-Iron** [reply]
Ciao @snakuzzo , se possibile avrei bisogno un chiarimento... ho sistemato tutta la parte light, switch e binary sensor.
Ho però una domanda per la parte Sensor. ho dei sensori che ho creato nel seguente modo
```

  - platform: mqtt
    name: "Ass Lavastoviglie"
    state_topic: "shellies/shellyplug-s-xxxxxx/relay/0/power"
    value_template: "{{ value }}"
    unit_of_measurement: "W"
  - platform: integration
    source: sensor.ass_lavastoviglie
    name: "Consumo Lavastoviglie"
    unit_prefix: k
    round: 2
  - platform: mqtt
    name: "Temp Sensore PlugS Lavastoviglie"
    state_topic: "shellies/shellyplug-s-xxxxxx/temperature"
    availability_topic: "shellies/shellyplug-s-xxxxxx/online"
    value_template: "{{ value }}"
    unit_of_measurement: "°C"
    payload_available: "true"
    payload_not_available: "false"
```
Per non lasciare tutto in configuration.yaml, ho creato un sidecar x mqtt. Vorrei sapere se la parte - platform: integration - posso incollarla all'interno di questo file o se devo lasciarla nel sidecar sensor.
spero di essere stato chiaro :-)

Grazie
